Doppler.Currency.Test/Integration/CurrencyControllerTests.cs

```diff
@@ -52,9 +50,12 @@ public async Task GetCurrency_ShouldBeHttpStatusCodeOk_WhenDateAndCurrencyCodeAr
             // Assert
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
             Assert.NotEmpty(responseString);
-            Assert.Contains(dateTime, responseString);
-            Assert.Contains("30", responseString);
-            Assert.Contains("10", responseString);
+
+            var dateString = DateTime.Parse(dateTime).ToUniversalTime();
```

Comment:
In my opinion, the expected value should be hardcoded.

Reply:
Done

Reply:
❤️